### ISSUE TITLE
Unable to change profile picture

### ISSUE THREAD

**Issue by paulabaker** [mgp25/Instagram-API#3068]
# Bug Report

---

### Notes:
We will close your issue, sometimes without answering, if any of the following are met:
* You have not included your code or your debug log
* You are asking about `challenge_required`, `checkpoint_required`, `feedback_required` or `sentry_block`. They've already been answered in the Wiki and *countless* closed tickets in the past!
* You have used the wrong issue template
* You are posting screenshots, which are too painful to help you with 

Please make sure you have done the following before submitting your issue:
* You have [Searched](https://github.com/mgp25/Instagram-API/search?type=Issues) the bug-tracker for similar issues including **closed** ones
* You have [Read the FAQ](https://github.com/mgp25/Instagram-API/wiki/FAQ)
* You have [Read the wiki](https://github.com/mgp25/Instagram-API/wiki)
* You have [Reviewed the examples](https://github.com/mgp25/Instagram-API/tree/master/examples)
* You have [Installed the api using ``composer``](https://github.com/mgp25/Instagram-API#installation)
* You are [Using latest API release](https://github.com/mgp25/Instagram-API/releases)

---

### Description
I'm unable to change the profile picture. I get "Invalid request options." as the error message.

### Code
Please post your code relevant to the bug in the section below:
```php
$username = '';
$password = '';
$file = '/usr/tmp/sg.jpg';
$ig = new Instagram($igDebug, $igTruncatedDebug, $storageConfig);
$ig->login($username, $password);

$photo = new \InstagramAPI\Media\Photo\InstagramPhoto($file);
$response = $ig->account->changeProfilePicture($photo->getFile());
``` 

### Debug Log
Please post your debug log in the section below; You can enable the debug log by using `new Instagram(true)` instead of `new Instagram()`. Additionally, you can do `\InstagramAPI\Debug::$debugLog = true;` for the debug log to be saved to a file:
```php
POST:  https://i.instagram.com/api/v1/accounts/change_profile_picture/
→ 33.54kB
← 400    58B
RESPONSE: Something is wrong. Please check your input and try again.
```

**Comment by martsdev** [comment]
Same [error](https://github.com/mgp25/Instagram-API/issues/3067)

**Comment by florianmarical** [comment]
Apparently, Instagram has stopped this feature since 30th October.

See : #2971

**Comment by Nestrot** [comment]
> Apparently, Instagram has stopped this feature since 30th October.

No. This is a bug that has been fixed for premium subscription.

**Comment by florianmarical** [comment]
sorry, I have just see that feature use : 
```php
return $this->ig->request('accounts/change_profile_picture/')
```

**Comment by shadowgroundz** [comment]
> Apparently, Instagram has stopped this feature since 30th October.
> 
> See : #2971

Anyway I have some problem same with #2971  but, everytime I post a question author or collaborator always remove my issue. Is it possible to fix in next update for free user?
